### DISCUSSION TITLE
AttributeError: 'Trainer' object has no attribute 'lr_find'

### DISCUSSION THREAD

**Discussion by ozgugoksu** [Lightning-AI/pytorch-lightning#12043] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
6 t = pl.Trainer(gpus=[0])
----> 7 lr_finder = t.lr_find(module)

When I try to train a model, I got an error like;

**"AttributeError: 'Trainer' object has no attribute 'lr_find' "**

How can I fix this?
Thanks,

**Comment by rohitgr7** [comment, marked as answer]
hey @ozgugoksu 

its `trainer.tuner.lr_find`.

**Comment by BentleyOph** [comment]
Im still getting an attribute error after this..what could be the issue

**Reply by akihironitta** [reply]
@BentleyOph `Tuner` has been decoupled from `Trainer` since 2.0. Please see the release note or the PR:
- https://lightning.ai/pages/releases/2.0.0/
- https://github.com/Lightning-AI/lightning/pull/16462
